src/pages/components/Theme.qml

```diff
@@ -0,0 +1,18 @@
+/****************************************************************************
+**
+** Copyright (C) 2015 Jolla Ltd.
+** Contact: Raine Makelainen <raine.makelainen@jolla.com>
+**
+****************************************************************************/
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+pragma Singleton
+import QtQml 2.2
+
+QtObject {
+    property color backgroundColor: "black"
+    property real backgroundOpacity: 0.7
```

Comment:
Having a singleton Theme for Browser is a bit over kill as Background.qml can handle both of these.

What do you think?